processout-sdk/src/androidTest/java/com/processout/processout_sdk/UITestSuite.java

```diff
@@ -34,10 +34,20 @@
 @LargeTest
 public class UITestSuite {
 
-    private String projectId = "test-proj_gAO1Uu0ysZJvDuUpOGPkUBeE3pGalk3x";
-    private String privateKey = "key_sandbox_mah31RDFqcDxmaS7MvhDbJfDJvjtsFTB";
+    private static final String projectId = "test-proj_gAO1Uu0ysZJvDuUpOGPkUBeE3pGalk3x";
+    private static final String privateKey = "key_sandbox_mah31RDFqcDxmaS7MvhDbJfDJvjtsFTB";
     private Gson gson = new GsonBuilder().disableHtmlEscaping().create();
 
+    private static final int cardExpirationYear;
+    private static final int cardExpirationMonth;
+
+    static {
+        // card is set to expire in one year from now
+        Calendar calendar = Calendar.getInstance();
+        cardExpirationYear = calendar.get(Calendar.YEAR) % 100 + 1;
+        cardExpirationMonth = calendar.get(Calendar.MONTH);
+    }
+
```

Comment:
The tests in this class were failing because the expiration date was hardcoded to October 2020.
This refactor sets the card expiration date to one year from now.